proto/ligato/linux/interfaces/models.go

```diff
@@ -315,4 +314,5 @@ func (m *Interface) MarshalJSON() ([]byte, error) {
 // UnmarshalJSON ensures that field of type 'oneOf' is correctly unmarshaled
 func (m *Interface) UnmarshalJSON(data []byte) error {
 	return jsonpb.UnmarshalString(string(data), m)
-}
\ No newline at end of file
+}
+*/
```

Comment:
This looks similar to the other commented-out marshaling/unmarshaling. This time with reason why it exists. Is this still needed? Does it explain the reason for the Marshalling my other comment?
It probably should be removed if not needed or reimplemented using protov2 tooling if it is actually needed.